src/Products/Common/Resources/Resources.cs

```diff
@@ -43,41 +43,45 @@ public static string GetFreeFileName(string directory, string fileName)
             {
                 throw e;
             }
+
             return resultFileName;
         }
 
         /// <summary>
-        /// Generate exception
+        /// Generate exception.
         /// </summary>
-        /// <param name="ex">Exception</param>
-        /// <returns>ExceptionEntity</returns>
+        /// <param name="ex">Exception.</param>
+        /// <returns>ExceptionEntity.</returns>
         public ExceptionEntity GenerateException(System.Exception ex)
         {
             // Initiate Exception entity
             ExceptionEntity exceptionEntity = new ExceptionEntity();
+
             // set exception data
             exceptionEntity.message = ex.Message;
             exceptionEntity.exception = ex;
             return exceptionEntity;
         }
 
         /// <summary>
-        /// Generate exception for password error
+        /// Generate exception for password error.
         /// </summary>
-        /// <param name="ex">Exception</param>
-        /// <param name="password">string</param>
-        /// <returns>ExceptionEntity</returns>
-        public ExceptionEntity GenerateException(System.Exception ex, String password)
+        /// <param name="ex">Exception.</param>
+        /// <param name="password">string.</param>
+        /// <returns>ExceptionEntity.</returns>
+        public ExceptionEntity GenerateException(System.Exception ex, string password)
```

Comment:
Codacy found an issue: [Make 'GenerateException' a static method.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5850423)